symplyphysics/laws/astronomy/lifetime_of_star_on_main_sequence.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The main sequence is the stage of stellar evolution formed by the stars at this stage and the corresponding luminosity class.
+## The main sequence of stars falls after the protostar stage. At this point, the age of the star is considered zero and it is
```

Comment:
`The main sequence of stars falls`
What do you mean? Why and where does it fall?